api/api.proto

```diff
@@ -1033,6 +1033,31 @@ service OpenStorageCloudBackup {
         body: "*"
       };
     }
+
+  // Create Schedule
+  rpc SchedCreate(SdkCloudBackupSchedCreateRequest)
```

Comment:
I am of the opinion that these three functions should be in separate service , so as to achieve consistency with function names such as Create instead of SchedCreate and the same with others

Reply:
@lpabon what is your opinion on this, I'm fine with both approaches. I also have local commit for separate service for cloud backup schedule API's.